### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by IAmJohnOconnell**
I can almost guarantee that there is going to be some extra code in there. However, Upon some testing I believe it to be a viable solution to a landing page, and the logo refresh issue.

### CODE REVIEW

**Comment by RafGDev** [comment]
is the content supposed to be visible in the '/' directory? There doesn't seem to be anything.

**Comment by RafGDev** [comment]
I'll leave this to @joedaouddotcom to review

**Comment by JGefroh** [comment]
Looks like it's missing some of the latest bug fixes from `dev`. Fetch the latest `dev` changes and merge `dev` into this branch. Once the other issues are resolved, it'll be good to go.

File: frontend/app/modules/sample/sample-module.js [hunk 1]
```diff
@@ -15,6 +15,15 @@
           templateUrl: 'sample-2.html',
           controller: 'SampleController',
           controllerAs: 'vm'
+      }).state('chat-app.landing', {
+        url: '/landing',
+        templateUrl: 'landingPage.html',
+        controller: 'SampleController',
+        controllerAs: 'vm'
+      }).state('chat-app', {
```
Comment (JGefroh) [inline frontend/app/modules/sample/sample-module.js]: The name of this state conflicts with the root state defined in `app.js`. This state is actually not needed - you can get rid of it safely.

File: frontend/app/modules/sample/sample-module.js [hunk 2]
```diff
@@ -15,6 +15,15 @@
...
+        url: '/landingPage',
+        templateUrl:'landingPage.html',
+        controller: 'SampleController',
+        controllerAs: 'vm'
       });
```
Comment (JGefroh) [inline frontend/app/modules/sample/sample-module.js:23]: Missing a closing `}`.

File: frontend/app/modules/sample/sample-module.js [hunk 3]
```diff
@@ -15,6 +15,15 @@
           templateUrl: 'sample-2.html',
           controller: 'SampleController',
           controllerAs: 'vm'
+      }).state('chat-app.landing', {
+        url: '/landing',
```
Comment (JGefroh) [inline frontend/app/modules/sample/sample-module.js]: You can set this url to `'/'` to make this load as the default route.

**Comment by IAmJohnOconnell** [comment]
Updated files to reflect the above corrections.

**Pull request merged**